CRM/Extendedreport/Form/Report/ExtendedReport.php

```diff
@@ -2330,7 +2330,7 @@ function extractFieldsAndFilters($field, $fieldName, &$filter) {
             'Multi-Select',
             'AdvMulti-Select',
             'CheckBox',
-          ])
+          ] || $field[$htmlType] == 'Select' && $field['serialize'] == 1)
```

Comment:
Should this just be any serialized field?

Reply:
Oh yeah, I think that's right. And also I think this logic is wrong (it would need a parenthesis in there).  Working on fix...